vignettes/openxlsx2_formulas_manual.Rmd

```diff
@@ -121,51 +121,51 @@ wb <- wb_workbook()$add_worksheet()$
 #### `dataTable` formula differences ####
 |   |     A       |  B   |      C        |
 |---|-------------|------|---------------|
-| 1 | SalesPrice  | COGS | SalesQuantity |
+| 1 | sales_price | COGS | sales_quantity|
 | 2 |     20      |  5   |      1        |
 | 3 |     30      |  11  |      2        |
 | 4 |     40      |  13  |      3        |
 
-Given a basic table like the above, a similarly basic formula for "Total_Sales" would be "=A2 * C2" with the row value changing at each row.
+Given a basic table like the above, a similarly basic formula for `total_sales` would be  "total_sales = A2 * C2" with the row value changing at each row.
```

Comment:
See next!

Reply:
Just for reference, I assume that `"=A2 * C2"` is what is intended here. So that the quoted part is identical to a basic formula.

Reply:
Will change that! nice catch